doc/classes/Color.xml

```diff
@@ -218,12 +218,45 @@
 			<return type="Color" />
 			<argument index="0" name="rgba" type="String" />
 			<description>
+				Returns a new color from [code]rgba[/code], an HTML hexadecimal color string. [code]rgba[/code] is not case sensitive, and may be prefixed with a '#' character.
+				[code]rgba[/code] must be a valid three-digit or six-digit hexadecimal color string, and may contain an alpha channel value. If [code]rgba[/code] does not contain an alpha channel value, an alpha channel value of 1.0 is applied.
+				If [code]rgba[/code] is invalid a Color(0.0, 0.0, 0.0, 1.0) is returned.
+				[codeblocks]
+				[gdscript]
+				var green = Color.html("#00FF00FF") # set green to Color(0.0, 1.0, 0.0, 1.0)
+				var blue = Color.html("#0000FF") # set blue to Color(0.0, 0.0, 1.0, 1.0)
+				[/gdscript]
+				[csharp]
+				var green = Color.Html("#00FF00FF"); // set green to Color(0.0, 1.0, 0.0, 1.0)
+				var blue = Color.Html("#0000FF"); // set blue to Color(0.0, 0.0, 1.0, 1.0)
```

Comment:
No problem. Do you think we should remove the csharp example from this method, and improve/expand the relevant constructor method, or include it like your example with a note that it's not directly implemented in csharp?

Reply:
I like the idea of having the example with a note because some people may come here from a GDScript tutorial. I don't know if that's something we've done before, it might not follow our style.

cc @godotengine/documentation